### ISSUE TITLE
 Allow specific items two bypass

### ISSUE THREAD

**Issue by theISSUEfinder3** [Gnat008/PerWorldInventory#216]
the title says it all. And it coud be nive with a /perworldgive [world] [player] [item/ID] thanks

**Comment by dedounet** [comment]
Yes, like something earned in a world that you want to keep as a reward in the others!
